.github/workflows/ci.yml

```diff
@@ -0,0 +1,73 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+      - actions
+  pull_request:
+    types: [opened, synchronize]
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build:
+  
+    strategy:
+      matrix:
+        target:
+          - ubuntu-latest
+          - ubuntu-18.04
+          - windows-latest
+          - macos-latest
+        include:
+          - target: ubuntu-latest
+            artifact: endless-sky-editor
+            binary: endless-sky-editor
+          - target: ubuntu-18.04
+            artifact: endless-sky-editor
+            binary: endless-sky-editor
+          - target: windows-latest
+            artifact: release/endless-sky-editor.exe
+            binary: release/endless-sky-editor.exe
+          - target: macos-latest
+            artifact: endless-sky-editor.app/
+            binary: endless-sky-editor.app/Contents/MacOS/endless-sky-editor
+
+    runs-on: ${{ matrix.target }}
+
+    steps:
+
+    - uses: actions/checkout@v2
+
+    - name: Cache Qt
+      id: cache-qt
+      uses: actions/cache@v1
+      with:
+        path: ${{ github.workspace }}/Qt/
+        key: ${{ runner.os }}-QtCache
+
+    - name: Install Qt
+      uses: jurplel/install-qt-action@v2
+      with:
+        version: "5.15.2"
```

Comment:
Should this be a matrix parameter?

Reply:
As soon as we want it to differ between targets, yes. At the moment, probably not.